lib/puppet/configurer.rb

```diff
@@ -422,7 +422,10 @@ def send_report(report)
   end
 
   def save_last_run_summary(report)
-    mode = Puppet.settings.setting(:lastrunfile).mode
+    # Pass nil to use file system inherited permissions, set by the Puppet Agent package.
+    mode = Puppet::Util::Platform.windows? ?
+      nil :
+      Puppet.settings.setting(:lastrunfile).mode
```

Comment:
I thought maybe we should go back and change defaults.rb for this at https://github.com/puppetlabs/puppet/blob/master/lib/puppet/defaults.rb#L1647

But then @glennsarti reminded me that we explicitly no longer manage modes on Windows through defaults.rb as a result of all the packaging refactoring / perm code we worked on.